zul/src/archive/web/js/zul/box/Splitter.js

```diff
@@ -397,10 +401,13 @@ zul.box.Splitter = zk.$extends(zul.Widget, {
 				runPrev.style.overflow = 'hidden';
 		}
 
-		if (w = run.nextwgt)
-			zUtl.fireSized(w, -1); //no beforeSize
-		if (w = run.prevwgt)
+		for (var i = 0, w; i < 2; i++) {
+			w = wgts[i];
+			if (w && flexReset[i]) {
```

Comment:
No `wgts` and `flexReset` if `if (runNext && runPrev)` block not being executed.